File: Python/tests/test_bart_par4_learning.py
import pytest

from hbayesdm.models import bart_par4_learning


def test_bart_par4_learning():
    _ = bart_par4_learning(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()

File: R/tests/testthat/test_bart_par4_learning.R
context("Test bart_par4_learning")
library(hBayesDM)

test_that("Test bart_par4_learning", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(bart_par4_learning(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})
